frontend/src/CourseCard.css

```diff
@@ -0,0 +1,47 @@
+.CourseCard {
+    display: flex;
+    background: #F6F8F5;
+    box-shadow: 0px 4px 25px rgba(0, 0, 0, 0.24);
+    border-radius: 15px;
+    font-family: 'Roboto', sans-serif; 
```

Comment:
Looking at the design, I'd suggest setting additionally on CourseCard class:
```
max-width: 970px;
min-width: 700px;
```
This will hold the element as max 970px (as design specifies) and also allows for some resizing for smaller screens if needed. There might be some priority for the textbox, so the min value is arguable, I guess :)

Reply:
added, thank you! :)